generated/nidaqmx/_library_interpreter.py

```diff
@@ -6386,14 +6387,16 @@ def read_analog_waveform(
             timeout,
             FillMode.GROUP_BY_CHANNEL.value,
             waveform.raw_data,
-            [waveform.extended_properties]
+            [waveform.extended_properties],
```

Comment:
Don't allocate a list when extended properties are disabled. Pass None and update the type hint to allow None.